webhookv3/event_data.go

```diff
@@ -0,0 +1,58 @@
+package webhookv3
```

Comment:
The structured events are in a separate file so that it is easier to review simple PRs that add new event types and/or fields without changing any cross-cutting functionality or behavior.